TopAppBar/index.d.ts

```diff
@@ -0,0 +1,31 @@
+import MaterialComponent from '../MaterialComponent';
+import {VNode} from 'preact';
+
+declare interface ITopAppBarProps extends JSX.HTMLAttributes {
+  short?: boolean;
+  'short-collapsed'?: boolean;
+}
+export default class TopAppBar extends MaterialComponent<ITopAppBarProps, {}> {
+  static Section: typeof Section;
+  static Icon: typeof Icon;
+  static Title: typeof Title;
+  static Row: typeof Row;
+}
+
+declare class Row extends MaterialComponent<{}, {}> {}
+
+declare interface ISectionProps extends JSX.HTMLAttributes {
+  'align-start'?: boolean;
+  'align-end'?: boolean;
+}
+declare class Section extends MaterialComponent<ISectionProps, {}> {}
+
+declare interface IIconProps extends JSX.HTMLAttributes {
+  menu?: boolean;
+}
+declare class Icon extends MaterialComponent<IIconProps, {}> {}
+
+declare interface ITitleProps extends JSX.HTMLAttributes {
+  title?: string;
```

Comment:
Where is `title` defined in the [documentation](https://github.com/material-components/material-components-web/tree/master/packages/mdc-top-app-bar)? Should it even be here?